CMakeLists.txt

```diff
@@ -28,7 +29,10 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 # Ensure that we don't accidentally install into /usr/etc
-set(CMAKE_INSTALL_FULL_SYSCONDIR "/etc")
```

Comment:
The variables `CMAKE_INSTALL_XXX` and `CMAKE_INSTALL_FULL_XXX` are part of [GNUInstallDirs](https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html) cmake module (included above). They are is used produce interoperable install destinations please have a look.

Reply:
You want to keep CMAKE_INSTALL_FULL_SYSCONDIR?

Reply:
You must set `CMAKE_INSTALL_FULL_SYSCONDIR` as earlier. I'm okay with creating new variables if you need shorthands.